Install-Guide/Community_Packages.md

```diff
@@ -0,0 +1,36 @@
+### Community Packages
+
+An **X** means packages are planned to be in the repository.
```

Comment:
This (line 3 to 6) is rendered as a single line. You could either convert these to a markdown list or add two or more whitespaces at the end which will be rendered as linebreak.